ArduPlane/GCS_Mavlink.cpp

```diff
@@ -884,6 +886,7 @@ GCS_MAVLINK_Plane::data_stream_send(void)
         if (plane.control_mode != MANUAL) {
             send_message(MSG_PID_TUNING);
         }
+        send_message(MSG_LANDING);
```

Comment:
I debated, I was leaving the control of it within the AP_Landing library though, as I could see landing wanting to get one last cancellation message out on mode change. Can make the change if you prefer though since nothing currently does what I described.